apis/apis.go

```diff
@@ -40,6 +43,9 @@ func fixUsername(username string) string {
 }
 
 func (a *AnalysisAPI) RunningAnalyses(ctx context.Context, username string, limit int) (*AnalysisListing, error) {
+	ctx, span := otel.Tracer(otelName).Start(ctx, "RunningAnalyses")
+	defer span.End()
+
```

Comment:
Anything like this that sets off a span, checking that I named them something that seems accurate (usually, the function they're at the top of) is all the review that's needed.